notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        !optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 {model_dir}
</pre><p>suggest do not expose task at all or at least use more generic text-to-image instead of stable-diffusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>could you please use optimum_cl helper? <a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/cmd_helper.py#L26" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/cmd_helper.py#L26</a></p>

Reply:
fixed